README.md

```diff
@@ -45,7 +45,7 @@ $ cp $(i18n-tasks gem-path)/templates/minitest/i18n_test.rb test/
 
 ## Usage
 
-Run `i18n-tasks` to get the list of all the tasks with short descriptions.
+Run `i18n-tasks` or `bundle exec i18n-tasks` (recommended) to get the list of all the tasks with short descriptions.
```

Comment:
Let's just say `bundle exec i18n-tasks`.

Reply:
Done.

Reply:
The rest of the document still refers running it without the `bundle exec` but I'm not sure if it's a good idea to replace it everywhere.